### PR TITLE
Edits to Change Data Capture

### PR DESCRIPTION
**Pull request by graciegoheen**
## Description & motivation

Closes issue #1729 and removes old version of blog post.

@joellabes Open to feedback if this doesn't fix the issues you brought up!

### CODE REVIEW

**Comment by jasnonaz** [comment]
looks great! Can we also add a redirect in the `_redirect` file? Think it would look like `blog/change-data-capture-metrics	blog/change-data-capture	301` - that way any existing references to the other url get redirected over to the new one.

**Review by jasnonaz — APPROVED** [review]
Good to merge once we get the redirect in

File: website/blog/2022-07-12-change-data-capture.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ Her goal is to capture **all** versions of the `fct_income` model for January. S
 | January 2022 | electronics | 152 | 02/10/22 08:00:00 |
 | January 2022 | books | 202 | 02/10/22 08:00:00 |
 
-In order to achieve this **long table of history**, she decides to start [snapshotting](https://docs.getdbt.com/docs/building-a-dbt-project/snapshots) `fct_income`.
+In order to achieve this **long table of history**, she decides to do something that ⚠️ goes against dbt Lab's best practices ⚠️ - [snapshotting](https://docs.getdbt.com/docs/building-a-dbt-project/snapshots) her final model, `fct_income`.
```
Comment (joellabes) [inline website/blog/2022-07-12-change-data-capture.md]: I was thinking we should use a Docusaurus [admonition](https://docusaurus.io/docs/next/markdown-features/admonitions), to get some visual callout for anyone who is just skimming. Do you think it's overkill? 

```suggestion
In order to achieve this **long table of history**, she decides to start [snapshotting](https://docs.getdbt.com/docs/building-a-dbt-project/snapshots) her final model, `fct_income`.

:::caution Don't be like Joanne

I'm including the code samples for completeness, but remember: the method described in this scenario contradicts dbt best practices. Either of the [solutions](#solution-1-downstream-incremental-model) [detailed](#solution-2-upstream-snapshots) later in the article is a better approach.

:::
```

**Review by joellabes — COMMENTED** [review]
Have added an example of the sort of callout I had imagined - I think that putting disclaimers in the prose is going to get swallowed up unless you're carefully reading it all (as evidenced by the fact that _I_ missed them on the first pass)

**Comment by graciegoheen** [comment]
I like it @joellabes - just added that change, let me know what you think!

**Review by joellabes — APPROVED** [review]
Love it 👏 let's rumble 🚢

**Pull request merged**